README.md

```diff
@@ -0,0 +1,27 @@
+# Half-Life SDK for Xash3D
+
+Half-Life SDK for Xash3D with some fixes.
+
+## How to build
+
+### Windows
+
+TODO
+
+### Linux
+
+TODO
+
+### OS X
+
+TODO
+
+### FreeBSD
+
+```
+    cd dlls
+    gmake CXX=clang++ CC=clang
```

Comment:
@nekonomicon cc is clang for me.
I don't actually understand how CC in makefile is supposed to be set to clang automatically. There's no even such environment variable by default.

g++ link appeared after I upgraded gcc.

Anyway I would prefer to set compilers explicitly and clang is probably better choice on FreeBSD, that's why I wrote it in README.md